CHANGELOG.md

```diff
@@ -2029,3 +2029,34 @@ This is a full list of proofs that have changed form to use irrelevant instance
   Inverse⇒Injection : Inverse S T → Injection S T
   ↔⇒↣ : A ↔ B → A ↣ B
   ```
+
+* Added new isomorphisms to `Data/Fin/Properties`:
```

Comment:
Should use `.` rather than `/` in the module names here?

Reply:
I have already changed `/` for `.`